Error: Use `aria-describedby` to reference a group description.
Error: For broadest accessibility, implicitly associated `label` elements should also be explicitly associated with their control, using `for`.
Error: Checkboxes in a group should use a `name` value that ends with "[]" to allow server-side code to collect the values as an array (or similar).
diff --git a/output/Checkbox Group with Accessible Description/2/b63ae339-7af4-44c1-a799-9bb7c7430e2f.html b/output/Checkbox Group with Accessible Description/2/b63ae339-7af4-44c1-a799-9bb7c7430e2f.html
index 7643163..1c8bbc7 100644
--- a/output/Checkbox Group with Accessible Description/2/b63ae339-7af4-44c1-a799-9bb7c7430e2f.html	
+++ b/output/Checkbox Group with Accessible Description/2/b63ae339-7af4-44c1-a799-9bb7c7430e2f.html	
@@ -1,7 +1,16 @@
-<fieldset>
+<fieldset aria-describedby="fruit-desc">
   <legend>What fruits do you like?</legend>
-  <p>Note: You can change this later</p>
-  <label><input type="checkbox" name="fruits" value="bananas"> Bananas</label>
-  <label><input type="checkbox" name="fruits" value="oranges"> Oranges</label>
-  <label><input type="checkbox" name="fruits" value="apples"> Apples</label>
+  <p id="fruit-desc">Note: You can change this later</p>
+  <label for="bananas">
+    <input type="checkbox" name="fruits[]" id="bananas" value="bananas">
+    Bananas
+  </label>
+  <label for="oranges">
+    <input type="checkbox" name="fruits[]" id="oranges" value="oranges">
+    Oranges
+  </label>
+  <label for="apples">
+    <input type="checkbox" name="fruits[]" id="apples" value="apples">
+    Apples
+  </label>
 </fieldset>
\ No newline at end of file
